repost/api/routes/users.py

```diff
@@ -21,22 +21,25 @@
 
 
 @router.get('/me', response_model=User,
-            responses={HTTP_403_FORBIDDEN: {'model': ErrorResponse}})
+            responses={HTTP_400_BAD_REQUEST: {'model': ErrorResponse},
+                       HTTP_401_UNAUTHORIZED: {'model': ErrorResponse}})
 async def get_current_user(current_user: models.User = Depends(resolve_current_user)):
     """Get the currently authorized user."""
     return current_user
 
 
 @router.patch('/me', response_model=User,
-              responses={HTTP_403_FORBIDDEN: {'model': ErrorResponse}})
+              responses={HTTP_400_BAD_REQUEST: {'model': ErrorResponse},
+                         HTTP_401_UNAUTHORIZED: {'model': ErrorResponse}})
 async def edit_current_user(*, current_user: models.User = Depends(get_current_user), edited_user: EditUser,
                             db: Session = Depends(get_db)):
     """Edit the currently authorized user."""
     return crud.update_user(db, username=current_user.username, **edited_user.dict(exclude_unset=True))
 
 
 @router.delete('/me',
-               responses={HTTP_403_FORBIDDEN: {'model': ErrorResponse}})
+               responses={HTTP_400_BAD_REQUEST: {'model': ErrorResponse},
+                          HTTP_401_UNAUTHORIZED: {'model': ErrorResponse}})
 async def delete_current_user(current_user: models.User = Depends(get_current_user), db: Session = Depends(get_db)):
     """Delete the currently authorized user."""
     crud.delete_user(db, username=current_user.username)
```

Comment:
I am not entirely sure on how we set users up, but I was wondering if we should have 403 Error Response in case a user tries to edit or delete another user, if that is even possible.